code/game/objects/items/cards_ids.dm

```diff
@@ -206,6 +206,9 @@
 		user.visible_message("<span class='notice'>[user] shows you: [icon2html(src, viewers(user))] [src.name].</span>", "<span class='notice'>You show \the [src.name].</span>")
 	add_fingerprint(user)
 
+/obj/item/card/id/proc/set_access(list/new_access_list, mode = ERROR_ON_FAIL)
+	return
```

Comment:
theres isn't any override to this so set_access always just return

Reply:
This is me trying to bypass TG's ID system

Reply:
3a8fa55